VaporConfig.h.in

```diff
@@ -0,0 +1,6 @@
+#ifndef VAPOR_CONFIG
+#define VAPOR_CONFIG
+
+#cmakedefine USE_OMP
```

Comment:
There is already a header for the CMake configuration: `include/vapor/CMakeConfig.h `

Reply:
Moving the content of this file to `include/vapor/CMakeConfig.h` seems to break the compilation, unless there's a smarter way that I'm not aware of.

Reply:
If that is the case, you can replace this file with `target_compile_definitions (vdc PUBLIC USE_OMP)`

Reply:
I don't quite understand, do you mean replace the content of this file to be the line you suggested, or delete this file and introduce a new file with the line you suggested, or something else? 

If you know how to do it and it's not a big workload, could you go ahead and make the changes?

Reply:
Or maybe we just keep this approach -- I don't really see any adverse effect with it. If you're aware of any pros and cons, please let me too!

Reply:
It's not major but it adds unnecessary complexity and redundant CMake includes. I replaced it with my suggested solution.